src/docs/database-migrations.mdx

```diff
@@ -60,6 +60,33 @@ Always commit the changes to `migrations_lockfile.txt` with your migration.
 
 There are some things we need to be careful about when running migrations.
 
+### Testing
```

Comment:
I put this at the top of the "Guidlines" section because it seems like the most important guideline to me, but I'm open to moving it further down.